ports/geckolib/gecko_bindings/structs_debug.rs

```diff
@@ -2792,7 +2753,7 @@ impl ::std::clone::Clone for nsIRequest {
 #[repr(C)]
 #[derive(Debug, Copy)]
 pub struct EventStates {
-    pub mStates: ::std::os::raw::c_ulong,
+    pub mStates: ::std::os::raw::c_ulonglong,
```

Comment:
It is on mine, at least, so I'm going to assume this is change is OK.  (And in fact better than using `long`, which won't be big enough on 32 bit machines.)